### DISCUSSION TITLE
30 个字介绍一个项目，介绍被选中后会获得一个 NFT

### DISCUSSION THREAD

**Discussion by luckyyang** [rebase-network/hello-world#107] [General]
看到[一篇文章](https://web3caff.com/zh/archives/7342)，谈到了应该找到简单的几句话介绍一个项目，其中有几个例子:

- Helium：从根本上降低了建立物理 WAP 网络的成本结构；
- Solana：技术的可扩展性让生态可扩展；
- Graph：GraphQL 可用于抽象检索，而高效的分散读取对大型应用很重要；
- Fractal：NFT 游戏市场潜力巨大，团队实力足够强；
- LayerZero：可以有效优化当下跨链非常复杂的状况；
- Arweave：永久存储是一种全新的基础技术，可以释放出我们无法想象的用例；
- Superfluid：解决支付问题的最佳解决方案；
- Serum：CLOB 是处理价格发现问题的最佳解决方案；
- Ceramic：可靠的去中心化数据库，可以为 dTwitter 或 dReddit 等应用提供动力；
- Fluence：dAWS Lambda；
- UXD：创建完全抵押的去中心化稳定币的最有效的资本运作方式。

这里征集一下大家对各个项目的介绍，要求是 **30 个字以内**。另外请附上项目链接。直接在下面回复即可。

介绍被选中后会获得一个 NFT，请各位评论的时候附上自己的 CKB 钱包地址，钱包地址可以使用 https://flashsigner.com/ 注册

被选中的会放到这个 repo: https://github.com/rebase-network/web3-projects-in-30-words/blob/main/README.md

**Comment by NftTopBest** [comment]
Web3 NFT Browser: 给小白用户使用的 Web3 世界入口
ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsqvcxfveauw2vqh6tt0qp8xfuxmjed469cq6nr65a

**Comment by arc0035** [comment]
Nym: 带激励版的Tor，使用密码学技术确定路由的服务质量，并予以奖励；

Arbitrum：交互式Rollup，双方通过二分法将分歧缩小为一个指令，供合约执行并论断。

ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsqd5w0z4zvlxne6xk9w6jrvsvllk62lut9qdwywqq

**Comment by carrot93** [comment]
NFT  Value Assessment Plan: 拟合评估算法，根据综合指数、趋向图、共识、稀缺程度划分等级。

ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsqtrj80t8daee2l0v7g4z3heavpt9red5asfk9h47

**Comment by spark8899** [comment]
defi new world: 新时代defi交易
ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsqg7hakvzhr604rl6x92ksjqt5e5dmuawfs2x73vn

**Comment by luckyyang** [comment]
@carrot93 @arc0035 @NftTopBest NFT已发送，请在 https://rostra.xyz/mynfts 查看
![2022-06-18 at 11 51](https://user-images.githubusercontent.com/1963646/174421771-abf639d6-dcc7-4257-9821-4d42c92b16c6.png)
